apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClientImpl.RoutineCrud.cs

```diff
@@ -0,0 +1,273 @@
+﻿// Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
This is the old copyright license - the new one uses https in the Apache link, and Google LLC instead of Google Inc.
(Please check for all new files.)

Reply:
Done.